docs/development_environment.mdx

```diff
@@ -40,7 +40,7 @@ In the future, if you want to get back to your development environment: open Vis
 
 ### Tasks
 
-The devcontainer comes with some useful tasks to help you with development, you can start these tasks by opening the command palette and select `Tasks: Run Task` then select the task you want to run.
+The devcontainer comes with some useful tasks to help you with development, you can start these tasks by opening the command palette with `Shift`+`Command`+`P`(Mac) / `Ctrl`+`Shift`+`P` (Windows/Linux) and select `Tasks: Run Task` then select the task you want to run.
```

Comment:
It's a slippery slope of we start documenting how vscode works instead of just documenting our dev container.

Reply:
I had doubt if we should add it, but helping others this question somehow pops up very often. We can remove or change it in the future if we see it is going in this direction. 

In general onboarding for new developers is a little bit hard (e.g. they may be experienced Python developers but work with another IDE and not familiar with our dev container or vscode tasks) so if we can make it easier for them I think we should add.

Reply:
Sure, but if I Google for `vscode command palette` Google will immediately tell me the keyboard shortcut that we document here. If we start writing the documentation aimed at a developer that should never need to look up any terms or Google a bit, I think we're doing a disservice to the developer. You learn better by doing a bit of investigating on your own.

Reply:
I don't think this slope is that slippery at all.

It is just some practical information that saves people some time and thereby makes it easier to start developing (which is the purpose of the page).

> Sure, but if I Google for `vscode command palette` Google will immediately tell me the keyboard shortcut that we document here. 

Using google is not the problem here. The problem is that the first (reasonable) action is to look for a (non-existing) button to open the command palette. Then when you can't find the button, you look in the menu under `Go` and `Run` (since it is a _command_ pallete, right?). Again nothing. You realize vscode is way different than the IDE you normally use, and only then you google how to open this pallette thingy. 

> If we start writing the documentation aimed at a developer that should never need to look up any terms or Google a bit, I think we're doing a disservice to the developer. You learn better by doing a bit of investigating on your own.

The argument to leave out practical information for didactic purposes does not seem valid for me since googling "how can I open the visual studio code command pallete" taught me :
- It is not written `pallete` (it is either `pallette` or `palette`. I keep forgetting which one...)
- having this tiny bit of information in the docs would have saved me from all the hassle described above.

Are these the things I should have learned?😀

Reply:
As a developer you need to start learning the tool you're using. The documentation for the tool in this case already exist online. It's the same with the programming languages we use in the project. We don't document the languages. We document our use of the languages.

I think your description of what to try is perfectly fine. First you explore on your own and if you don't find what you're looking for you turn to Google. I don't see that as a hassle, it's learning.